bake

```diff
@@ -331,17 +331,26 @@ function bake_cd () {
 function bake_find_bakefile_impl () {
   local project_root
   project_root="$(pwd)"
-
-  if [ -e "Bakefile" ]; then
+  if [ -f "Bakefile" ]; then
     echo "$project_root/Bakefile"
     return 0
   fi
 
-  if [ -e "bakefile" ]; then
+  if [ -f "bakefile" ]; then
     echo "$project_root/bakefile"
     return 0
   fi
```

Comment:
@kyleburton - this change forces `bakefile` to be a file (I think). Trying to force a differentiation from the bake dir.